classes/class-nets-easy-checkout.php

```diff
@@ -78,6 +79,7 @@ public function update_nets_easy_order( $cart ) {
 		$payment_id_session = WC()->session->get( 'dibs_payment_id' );
 
 		if ( $payment_id !== $payment_id_session ) {
+			nexi_terminate_session( $payment_id_session );
```

Comment:
Question: Should we not terminate both sessions in this case? Since the one from the form data will be incorrect after a reload anyway i believe.